### PR TITLE
added eth-utils

### PR DESCRIPTION
**Pull request by step21**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/eth-utils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  skip: true   # [py>=40 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools-markdown
+  run:
+    - cytoolz <1.0.0,>=0.10.1  # [py]
```
Comment (scopatz) [inline recipes/eth-utils/meta.yaml:24]: I think `py` is always true

Comment (step21) [reply recipes/eth-utils/meta.yaml:24]: so what is a selector that matches cpython but not pypy? (for the record, this is going off of setup.py, not tested myself so far)

Comment (step21) [reply recipes/eth-utils/meta.yaml:24]: ok found it 
```
# [python_impl == 'cpython']
# [python_impl == 'pypy'] ```

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**